packages/ui/src/components/BufficornGlobalData.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div class="stats-container" :class="{ even: index % 2 }">
+    <div class="stats">
+      <div class="stat-container" v-for="resource in stats" :key="resource.key">
+        <img class="icon" :src="imgUrl(resource.key)" alt="icon" />
+        <div class="stat">
+          <p class="key">{{ resource.key.toUpperCase() }}</p>
+          <p class="score">{{ resource.score }}</p>
+        </div>
+      </div>
+    </div>
+    <div class="small-title item user">
+      <p class="bufficorn-name">{{ name }}</p>
+    </div>
+    <img class="bufficorn-image" src="@/assets/bufficorn.svg" alt="Bufficorn" />
+  </div>
+</template>
+
+<script>
+import { useStore } from '@/stores/player'
+import { ref, computed } from 'vue'
+
+export default {
+  props: {
+    index: Number,
+    name: String,
+    vigor: Number,
+    speed: Number,
+    agility: Number,
+    coolness: Number,
+    select: {
+      type: Boolean,
+      default: false
+    }
+  },
+  setup (props) {
+    const player = useStore()
+    const stats = ref([
+      {
+        key: 'vigor',
+        score: props.vigor
+      },
+      {
+        key: 'speed',
+        score: props.speed
+      },
+      {
+        key: 'agility',
+        score: props.agility
+      },
+      {
+        key: 'coolness',
+        score: props.coolness
+      }
+    ])
+    const imgUrl = name => {
+      return `../src/assets/${name}.svg`
+    }
+    return { player, stats, imgUrl }
+  }
```

Comment:
```suggestion
```suggestion
    const makeAssetsUrl = assetName => `../src/assets/${assetName}.svg`
    
    return { player, stats, makeAssetUrl }
  }
```